docs/csharp/whats-new/csharp-8.md

```diff
@@ -438,3 +440,12 @@ var text = words[phrase];
 ```
 
 You can explore more about indices and ranges in the tutorial on [indices and ranges](../tutorials/ranges-indexes.md).
+
+## Enhancement of interpolated verbatim strings
+
+Order of the `$` and `@` tokens in [interpolated](../language-reference/tokens/interpolated.md) verbatim strings can be any:
+
+```csharp
+Console.WriteLine($@"Valid\in\C#6.0\and\later");  // output: Valid\in\C#6.0\and\later
+Console.WriteLine(@$"Valid\in\C#8.0\and\later");  // output: Valid\in\C#8.0\and\later
```

Comment:
This code sample doesn't benefit from interpolated strings in anyway. I suggest something like this (or any example that benefits from interpolation):
```csharp
var csharp = "C#";
Console.WriteLine($@"Valid\in\{csharp}6.0\and\later");  // output: Valid\in\C#6.0\and\later
Console.WriteLine(@$"Valid\in\{csharp}8.0\and\later");  // output: Valid\in\C#8.0\and\later
```

Reply:
Hm, I'm thinking about removing examples at all. At the end, it's only about the order of the first two tokens.